Sources/Sentry/SentryClient.m

```diff
@@ -36,6 +37,11 @@
 #import "SentryUser.h"
 #import "SentryUserFeedback.h"
 
+#if SENTRY_HAS_UIKIT
```

Comment:
Does that mean it does not work for macOS Apps?
If so, we need to document that.

Reply:
Please add to the property docs then.

Reply:
I think we should then add this also the `SentryOptions`.